alert/models.py

```diff
@@ -112,11 +124,11 @@ def resubscribe(self):
         return new_registration
 
 
-def register_for_course(course_code, email_address, phone):
+def register_for_course(course_code, email_address, phone, source='PCA', api_key=None):
```

Comment:
I think it's generally a best practice to define `"PCA"` as a constant and refer to that instead of using the string itself.